src/server/services/bodem/loodmetingen.ts

```diff
@@ -79,13 +79,13 @@ function transformLood365Response(response: Lood365Response): LoodMetingen {
             ? {
                 title: 'Rapport Lood in de bodem-check',
                 id: location.Workorderid!,
-                url: `${
-                  IS_ACCEPTANCE
-                    ? process.env.REACT_APP_BFF_API_URL_ACC
-                    : process.env.REACT_APP_BFF_API_URL
-                }${generatePath(BffEndpoints.LOODMETING_ATTACHMENTS, {
-                  id: location.Workorderid!,
-                })}`,
+                // https://acc.mijn.amsterdam.nl/lood-meting/undefined/services/lood/b203d335-9906-ee11-8f6e-0022489fda17/attachments
+                url: `${process.env.BFF_OIDC_BASE_URL}${generatePath(
+                  `${BFF_BASE_PATH}${BffEndpoints.LOODMETING_ATTACHMENTS}`,
+                  {
+                    id: location.Workorderid!,

```

Comment:
Is `Workorderid` wel eens null?

Reply:
Als `Reportavailable` dan moet er een workorderid zijn.

Reply:
Je zou dan ook `document: location.Workorderid ? { ... } : ` kunnen gebruiken, dat is veiliger. De `!` zouden we _eigenlijk niet_ moeten gebruiken als het niet hoeft.

Reply:
Paar extra checks toegevoegd zodat `!` niet meer nodig is.